restful_api/tests/factories.py

```diff
@@ -44,6 +45,7 @@ class EnrollFactory(factory.Factory):
     trainer_id = factory.SelfAttribute('trainer.id')
     has_passed = factory.LazyAttribute(lambda n: False)
     is_official = factory.LazyAttribute(lambda n: False)
+    created_timestamp = factory.LazyFunction(time.time)
```

Comment:
Ooo the result of this is 10-digit integer right? just cfming

Reply:
yep. here's an ss of the sample output printed

![Screenshot 2021-10-06 at 8 21 00 PM](https://user-images.githubusercontent.com/65717642/136201111-4253b144-1510-48bf-93f3-6b6f20d7d7e6.png)